pkg/restore/client.go

```diff
@@ -504,6 +504,7 @@ func (rc *Client) RestoreFiles(
 			return err
 		}
 	}
+	wg.Wait()
```

Comment:
This is a bugfix?

Reply:
yes, I think it fix the updateCh send on close channel at [L490](https://github.com/pingcap/br/blob/1cda453e3feb265b223eb032225254b1ff3d5ab8/pkg/restore/client.go#L490), when last file import success, if will wait until updateCh.Inc finish.